services/core-api/app/api/parties/party_appt/models/mine_party_appt.py

```diff
@@ -183,16 +166,16 @@ def find_by(cls,
                 cls.mine_party_appt_type_code.in_(mine_party_appt_type_codes))
         results = built_query.all()
 
-        if include_permittees and mine_guid:
+        if include_permit_contacts and mine_guid:
             #avoid circular imports.
             from app.api.mines.mine.models.mine import Mine
             mine = Mine.find_by_mine_guid(mine_guid)
             permit_permittees = []
             for mp in mine.mine_permit:
                 if not active_only:
-                    permit_permittees = permit_permittees + mp.permittee_appointments
+                    permit_permittees = permit_permittees + mp.permit_appointments
```

Comment:
Changing these two references of "permittee_appointments" to "permit_appointments" fixes the bug where the other permit-linked contacts were not being displayed on the Contacts page.